components/brave_rewards/browser/rewards_service_impl.h

```diff
@@ -74,6 +77,12 @@ class RewardsServiceImpl : public RewardsService,
                             public net::URLFetcherDelegate,
                             public base::SupportsWeakPtr<RewardsServiceImpl> {
  public:
+  enum ClearRewardsDataType {
```

Comment:
we don't need a second enum in rewards, we need a ledger enum so we can pass it through back to a ledger client method

Reply:
Moved to ledger